src/views-components/side-panel-button/side-panel-button.tsx

```diff
@@ -41,26 +43,33 @@ interface SidePanelState {
 
 type SidePanelProps = SidePanelDataProps & DispatchProp & WithStyles<CssRules>;
 
+const transformOrigin: PopoverOrigin = {
+    vertical: -50,
+    horizontal: 45
+};
+
+const checkButtonVisibility = ({ router }: RootState) => {
+    const pathname = router.location ? router.location.pathname : '';
+    const match = matchProjectRoute(pathname);
+    return match ? true : false;
```

Comment:
return match;

Reply:
@kowalczykpawel7 No!:) match can be object or null. My attr `showButton` has type `boolean`